lang/es.json

```diff
@@ -1,100 +1,100 @@
-            "toggleAudio": "Pause/Play Audio"
+            "close": "Cerrar presentación",
+            "update": "Actualizar posición y tamaño de la presentación",
+            "default": "Restablecer a la posición y tamaño predeterminada",
+            "toggleAudio": "Pausar/Reproducir sonido"
         },
         "introduceDialog": {
             "title": "Pres\u00E9ntame",
-            "flavorTitle": "Texto Adicional",
-            "colorsTitle": "Editar colores de Presentaci\u00F3n del Actor",
-            "durationTitle": "Duraci\u00F3n de la Presentaci\u00F3n",
-            "preview": "Vista Previa Privada",
-            "save": "Guardar Ajustes",
+            "flavorTitle": "Texto adicional",
+            "colorsTitle": "Editar colores de presentaci\u00F3n del actor",
+            "durationTitle": "Duraci\u00F3n de la presentaci\u00F3n",
+            "preview": "Vista previa privada",
+            "save": "Guardar ajustes",
             "introduce": "Presentar",
             "noneDefault": "El actor no tiene colores predeterminados",
             "default": "Reiniciar a predeterminados",
-            "actorColorSettings": "Ajustes de Color del Actor"
+            "actorColorSettings": "Ajustes de color del actor"
         },
         "settings": {
             "title": "Ajustes"
         },
         "colorSettings": {
-            "name": "Colores de la Presentaci\u00F3n",
-            "label": "Ajustes de Color",
+            "name": "Colores de la presentaci\u00F3n",
+            "label": "Ajustes de color",
             "default": "Reiniciar a colores predeterminados",
-            "preview": "Vista Previa Privada",
+            "preview": "Vista previa privada",
             "save": "Guardar",
             "actor": "Actor",
             "players": "Jugador",
             "friendly": "Amistoso",
             "neutral": "Neutral",
             "hostile": "Hostil",
             "template": "Plantilla",
-            "backgroundLabel": "Color de Fondo",
-            "textLabel": "Color de Texto",
-            "clipsLabel": "Mostrar Clips",
-            "audioLabel": "Audio",
-            "audioButton": "Configure sound or music",
+            "backgroundLabel": "Color de fondo",
+            "textLabel": "Color de texto",
+            "clipsLabel": "Mostrar clips",
+            "audioLabel": "Sonido",
+            "audioButton": "Configurar sonidos o música",
 	        "tester": "Prueba de Color"
         },
         "audioSettings": {
-            "title": "Audio Settings",
-            "loop": "Loop",
-            "audio": "Audio",
-            "volumeLabel": "Sound Volume",
-            "offset": "Start time",
-            "endOffset": "End Time",
+            "title": "Ajustes de sonido",
+            "loop": "Bucle",
+            "audio": "Sonido",
+            "volumeLabel": "Volumen del sonido",
+            "offset": "Tiempo de inicio",
+            "endOffset": "Tiempo de finalización",
             "fadeIn": "Fade in",
```

Comment:
Think you missed this one 😉 

'Fade in' refers to the audio fading in.

Reply:
This have a complicated translation in Spanish and almost everyone know the meaning in English.

Reply:
That's fair. The translation is 100% improved regardless. 👍